test/testdata/packager/visibility/hippanthropy/__package.rb

```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
+# typed: strict
+# enable-packager: true
+
+class Hippanthropy < PackageSpec
+  # this import will be okay because `Foo` includes `visible_to "tests"`
+  test_import Foo
+end
```

Comment:
Could we get a test that shows the opposite, like what the error would look like if it's violated?